projects/arrow/arrow/builders.py

```diff
@@ -190,6 +190,35 @@
     name='Install C++',
     workdir='cpp/build'
 )
+c_glib_meson = Meson(
+    args=[
+        'build',
+        '--prefix', util.Property('CMAKE_INSTALL_PREFIX'),
+        '--libdir', util.Property('CMAKE_INSTALL_LIBDIR'),
+        '-Dgtk_doc=true',
+    ],
+    workdir='c_glib'
+)
+c_glib_compile = Ninja(
+    j=util.Property('ncpus', 6),
+    name='Compile C GLib',
+    workdir='c_glib/build'
+)
+c_glib_install = Ninja(
+    'install',
+    name='Install C GLib',
+    workdir='c_glib/build'
+)
+c_glib_install_test_dependencies = Bundle(
```

Comment:
We can spare the test dependency installation time if we move `bundle` to the docker image, created issue https://github.com/ursa-labs/ursabot/issues/174